src/Core/Prefab/GlobeView.js

```diff
@@ -71,6 +71,7 @@ export const GLOBE_VIEW_EVENTS = {
     COLOR_LAYERS_ORDER_CHANGED,
 };
 
+let fullSizeDepthBuffer = null;
```

Comment:
You should store this in the GlobeView instance (prefixed with a `_` to mark it as private) because a user can create several GlobeViews (the same applies to PlaneView).